test/resource.test.js

```diff
@@ -0,0 +1,373 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const mongoose = require('mongoose');
+const {server} = require('../src/app');
+chai.should();
+chai.use(chaiHttp);
+
+let accessToken = null;
+
+describe("POST Tests Resource Account",()=>{
+    it("POST /Resource Account Bad Request", (done) =>{
+        chai.request(server).post("/resources/account")
+        .field('Content-Type', 'multipart/form-data')
+        .field('idAccount', '6164db5')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8292860)
```suggestion
        .field("idAccount", '6164db5')
```